app/controllers/user/shares_controller.rb

```diff
@@ -22,15 +22,16 @@ def new
              else
                Share.new
              end
-
-    @organisations = Organisation.all.order(:name)
+    @share.user = current_user
+    authorize @share
   end
 
   # POST /shares
   def create
     require_valid_params
 
     @share = create_share_with_auditing
+    authorize @share
```

Comment:
Note to self, this might be *after* it os created. Fix

Reply:
I don't think this line is necessary - it's authorized on line 52